embedded-graphics/src/primitives/mod.rs

```diff
@@ -30,42 +26,59 @@ pub trait Primitive: Dimensions {
 /// convenient macro.
 ///
 /// ```rust
-/// use embedded_graphics::pixelcolor::Rgb565;
-/// use embedded_graphics::prelude::*;
-/// use embedded_graphics::{egcircle, primitives::Circle};
-/// use embedded_graphics::style::{PrimitiveStyle, Styled};
-///
-/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> = egcircle!((10, 20), 30);
-/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> = egcircle!(Point::new(10, 20), 30);
+/// use embedded_graphics::{
+///     egcircle,
+///     pixelcolor::Rgb565,
+///     prelude::*,
+///     primitive_style,
+///     primitives::Circle,
+///     style::{PrimitiveStyle, Styled},
+/// };
+///
+/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> =
+///     egcircle!(center = (10, 20), radius = 30);
+/// let line_circle: Styled<Circle, PrimitiveStyle<Rgb565>> =
+///     egcircle!(center = Point::new(10, 20), radius = 30);
 /// let filled_circle: Styled<Circle, PrimitiveStyle<Rgb565>> = egcircle!(
-///     (10, 20),
-///     30,
-///     stroke_color = Some(Rgb565::RED),
-///     fill_color = Some(Rgb565::GREEN)
+///     center = (10, 20),
+///     radius = 30,
+///     style = primitive_style!(stroke_color = Rgb565::RED, fill_color = Rgb565::GREEN)
 /// );
 /// ```
 ///
-/// Style properties like `stroke_color` map to the properties in the [`PrimitiveStyle`] struct.
+/// Style properties like `stroke_color` map to methods on the [`PrimitiveStyleBuilder`] struct.
 /// For example, the following code makes two identical circles:
```

Comment:
The example creates 3 circles and not two as described. I would remove `circle_2` from the example to make the point clear.